docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md

```diff
@@ -120,7 +121,7 @@ Parameter | Type    | Description
 
 **Example**
 
-The following example demonstrates how to indent a summary information with 4 spaces.
+The following example demonstrates how to indent summary information by four spaces.
```

Comment:
yeah, it's usually recommended to spell out one through nine and use numbers in all other cases.
BTW, exceptions are:
* when two numerals < 10 and >= 10 are connected, as in ranges - in this case both numbers: `from 1 inch to 12 inches`
* when two numerals are used directly one after another: `52 twelve-inch displays`